hydrolib/core/basemodel.py

```diff
@@ -673,16 +674,17 @@ class FileModel(BaseModel, ABC):
     # Absolute anchor is used to resolve the save location when the filepath is relative.
     _absolute_anchor_path: Path = PrivateAttr(default_factory=Path.cwd)
 
-    def __new__(cls, filepath: Optional[Path] = None, *args, **kwargs):
+    def __new__(cls, filepath: Optional[Union[Path, str]] = None, *args, **kwargs):
```

Comment:
Consider a
```python
PathOrStr = Union[Path, str]
```
right above this class and use it in the other places

Reply:
What do you mean exactly?

Reply:
done